### PR TITLE
labwc.pot: update for new OmniPresent menu

### LINKED ISSUE #1311: Translations should be updated with the new OmniPresent menu
Translations (pot and po files) should be updated with the new OmniPresent menu: https://github.com/labwc/labwc/pull/1256

cc @bnason

### PR DESCRIPTION
**Pull request by 01micko**
closes #1311

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -362,6 +362,28 @@ translation strings under each English string.
...
 
+Code contributors may need to update relevant files if their additions
+affect UI elements (at the moment only `src/menu/menu.c`). In this case
+the `po/labwc.pot` file needs to be updated so that translators can
+update their translations. Remember, many translators are _not_ coders!
+
+The process is fairly trivial however does involve some manual steps.
+
+1. After adding and testing your code additions to satisfaction, backup
+`po/labwc.pot`. You need the custom header  from that file for the newly
```
Comment (Consolatis) [inline CONTRIBUTING.md:373]: Double space after `header`

**Comment by 01micko** [comment]
Whoops! Dark in here.

**Review by Consolatis — COMMENTED** [review]
Thanks as always. Nice addition to the docs :)

Minor spelling comment below. The commit itself also says `Omnipredent` rather than `OmniPresent`.

Other than that this LGTM.
We may want to change the menu item before 0.7.0 into something more easy to understand like "Visible on all workspaces" but we can do that outside of this PR.

**Comment by Consolatis** [comment]
:)

**Comment by 01micko** [comment]
> :)

:)

**Pull request merged**